src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,48 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder outputString = new StringBuilder("Report for period ")
```

Comment:
The semicolon is missing at the end of the statement. Ensure to terminate statements with a semicolon.